ownCloud/Resources/en.lproj/Localizable.strings

```diff
@@ -264,6 +264,8 @@
 
 "Forbidden Characters" = "Forbidden Characters";
 "File name cannot contain / or \\" = "File name cannot contain / or \\";
+"Item with same name already exists" = "Item with same name already exists";
+"An item with the same name already exists in this location." = "An item with the same name already exists in this location.";
```

Comment:
This items are missing:
Apply choice to all similar issues
Show all

Please check if all strings was added

Reply:
Thanks. Checked and "Apply to all" was also missing. Fixed all of them.